charts/kyuubi/templates/kyuubi-configmap.yaml

```diff
@@ -19,12 +19,26 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
it looks correct to me, cc @dnskr

Reply:
It is correct, but not required because the namespace of helm release is used by default. I mean I don't see any reasons or values by adding `namespace: {{ .Release.Namespace }}`.

Reply:
So, it's redundant, let's just keep it as-is to make thing simple.

Reply:
okay then we will keep it simple